cmd/restic/cmd_repair_packs.go

```diff
@@ -72,6 +72,7 @@ func runRepairPacks(ctx context.Context, gopts GlobalOptions, term *termstatus.T
 		if err != nil {
 			return err
 		}
+		defer f.Close()
```

Comment:
That change has no real effect as `defer` only runs once the surrounding function returns. And the operating system already closes all files once restic exists.